src/lib/github/addIssueToRepo.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ *  API Function that adds an issue with comments to a repository
+ */
+
+ export interface IssueWithComments {
+    title: string;
+    comments: string[];
+ }
+
+ export default async function addIssueToRepo(context: any): Promise<IssueWithComments[]> {
```

Comment:
function should take in `context` and an  array of the Issues with Comments that needs to get added to github. The return type should be [void](https://www.typescriptlang.org/docs/handbook/basic-types.html). i.e `Promise<void>`